ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockDataNodeId.java

```diff
@@ -56,6 +51,32 @@ private void populatePorts(ArrayList<Port> ports) {
     }
   }
 
+  /**
+   * Set the datacenters to which ssl is enabled. If the datacenter on which this datanode resides is
+   * part of the datacenters to which ssl is enabled, then this datanode will always return an SSL port in
```

Comment:
@nsivabalan yup - or some way for the clustermap to figure out which DC the user of the clustermap is in. This is pretty hard to do from what I can see of the code (and from @pnarayanan's comment above)  - but would have been a nice to have.